src/shogun/lib/auto_initialiser.h

```diff
@@ -124,14 +125,15 @@ namespace shogun
 								++idx;
 							}
 						}
-						width = GaussianKernel::to_log_width(linalg::median(result));
+						// width = 2 * (median-non-sqrt-dist ^ 0.5) ^ 2  = 2 * median-non-sqrt-dist
+						width = 2 * linalg::median(result);
```

Comment:
@karlnapf I think this is it now?

Reply:
result is the  median sq distance, then yet